### PR TITLE
Fixed issue #33, where an empty line was being displayed when no meta text was provided.

### LINKED ISSUE #33: Tooltip always includes meta text even when it's not present
In the current implementation, when the `meta` label is not defined, it'll add the meta span and a <br/> tag anyway, because the code that checks for the presence of `meta` is rewriting the variable:

``` javascript
01.  var meta = $point.getAttribute('ct:meta') || '';
02.  var value = $point.getAttribute('ct:value');

03.  if (options.tooltipFnc) {
04.    tooltipText = options.tooltipFnc(meta, value);
05.  } else {
06.    meta = '<span class="chartist-tooltip-meta">' + meta + '</span>';
07.    value = '<span class="chartist-tooltip-value">' + value + '</span>';

08.    if (meta) {
09.      tooltipText += meta + '<br>';
10.    } else {
(...)
```

When the code reaches line 8, `meta` will always be present, as it's now a string representing a `<span>` element. Because of this, the tooltip will always have a blank space on top of the value.

This doesn't happen then the `tooltipFnc` is used, but as this is not currently documented, I believe most people use the default behaviour, which is faulty.

This can easily be fixed by using a different variable name for the meta text and meta span string, or creating another variable like `hasMeta` to check if meta is present *before* changing it to a span.

I'll be glad to fork this and submit a PR with this change, if this is confirmed.

### PR DESCRIPTION
**Pull request by roooodcastro**
Fixed two minor spelling errors and fixed an issue with ct:meta where the plugin would always add the meta span to the label, along with a `<br/>`, even when no meta was provided. This resulted in an empty line at the top of the label. This issue didn't happen if the `options.tooltipFnc` was provided.

The fix was to add a variable called `hasMeta` that simply checked if the meta text is present, before wrapping it in a span. This fixes #33.

I also fixed the Jasmine's specs. Three specs were failing with the following error:

    TypeError: 'null' is not an object (evaluating '$point.parentNode.getAttribute') in file:///home/rodcastro/workspace/chartist-plugin-tooltip/src/scripts/chartist-plugin-tooltip.js (line 71) (1)

This happened due to the change introduced in the last commit. I added a check for the presence of `$point.parentNode` before calling `getAttribute` on it.

### CODE REVIEW

File: dist/chartist-plugin-tooltip.js [hunk 1]
```diff
@@ -84,7 +84,9 @@
           var $point = event.target;
           var tooltipText = '';
 
-          var meta = $point.getAttribute('ct:meta') || '';
+          var seriesName = ($point.parentNode) ? $point.parentNode.getAttribute('ct:series-name') : '';
```
Comment (roooodcastro) [inline dist/chartist-plugin-tooltip.js]: This check fixes the tests.

**Comment by Globegitter** [comment]
@roooodcastro Sorry for the delay - if you would update this branch to the latest changes from master (so I can auto-merge) would be happy to accept this PR.

**Comment by roooodcastro** [comment]
I just updated my branch with the latest changes from master :)

**Comment by Globegitter** [comment]
Thank you!

**Pull request merged**